crypto/pem/pem_pkey.c

```diff
@@ -88,7 +88,7 @@ EVP_PKEY *PEM_read_bio_PrivateKey(BIO *bp, EVP_PKEY **x, pem_password_cb *cb,
         PEMerr(PEM_F_PEM_READ_BIO_PRIVATEKEY, ERR_R_ASN1_LIB);
  err:
     OPENSSL_secure_free(nm);
-    OPENSSL_secure_clear_free(data, len);
```

Comment:
git blame names me as the author of this line.

Reply:
Looking at the history more closely, it looks like the "clear" got removed when the secure heap was used, and then you added a clear free back.  So yes, this should not be part of the code removal.  Thanks!